### PR TITLE
Trying to fix tests on windows

### PR DESCRIPTION
**Pull request by alexmojaki**

### CODE REVIEW

File: devtools/debug.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import os
 import sys
+from pathlib import Path
```
Comment (samuelcolvin) [inline devtools/debug.py:3]: see #49 & #50, for startup performance reasons we defer as many imports as possible.

Please can you move this import down into the function where it's called.

File: tests/test_expr_render.py [hunk 2]
```diff
@@ -1,40 +1,37 @@
 import asyncio
-import re
 import sys
 
 import pytest
 
 from devtools import Debug, debug
+from tests.utils import normalise_output
```
Comment (samuelcolvin) [inline tests/test_expr_render.py:7]:
```suggestion

from .utils import normalise_output
```

File: tests/test_main.py [hunk 3]
```diff
@@ -8,8 +8,7 @@
 
 from devtools import Debug, debug
 from devtools.ansi import strip_ansi
-
-pytestmark = pytest.mark.xfail(sys.platform == 'win32', reason='as yet unknown windows problem')
+from tests.utils import normalise_output
```
Comment (samuelcolvin) [inline tests/test_main.py:11]:
```suggestion

from .utils import normalise_output
```

**Review by samuelcolvin — COMMENTED** [review]
otherwise this looks great.

**Comment by samuelcolvin** [comment]
just noticed some tests are failing on windows, but CI is passing. We had the same problem on pydantic and discovered that CI setup like

```yaml
    - name: test with extras
      run: |
        make test
        coverage xml
```

when run with windows doesn't detect a failure in the first command.

I'll try to fix or skip these tests since they aren't really related to what you were fixing here (easier to fix here than on `main` since you've resolved a bunch of problems with windows :pray:)

**Comment by alexmojaki** [comment]
There's an actual failure in executing on Windows 3.9+. This is annoying and worrying. I've never seen `executing` affected by the OS. Any idea why it would matter? Isn't the bytecode the same?

**Comment by samuelcolvin** [comment]
Yes, just saw it.

I have absolutely no idea why this combination would fail. I also don't have a windows machine to test with.

Shall I leave this with you? I'm happy to skip or xfail this test on these os/version combinationd, but I understand you might want to look into it.

**Comment by alexmojaki** [comment]
I definitely need to investigate but for now please just xfail it.

**Comment by samuelcolvin** [comment]
thanks so much.

**Pull request merged**